symfony/ai-bundle/0.1/config/packages/ai.yaml

```diff
@@ -0,0 +1,23 @@
+# config/packages/ai.yaml
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+
+    Symfony\AI\Agent\Toolbox\Tool\Clock: null
+
+ai:
+    platform:
+        openai:
+            api_key: '%env(OPENAI_API_KEY)%'
+
+    agent:
+        default:
+            platform: 'symfony_ai.platform.openai'
+            model:
+                name: 'GPT'
+                version: 'gpt-4o-mini'
```

Comment:
I think this changed recently, but need to double check if already merged.

cc @welcoMattic

Reply:
I've commented a suggestion with the correct config